### DISCUSSION TITLE
arch qutebrowser started flickering for me roughly a week ago

### DISCUSSION THREAD

**Discussion by grault** [qutebrowser/qutebrowser#5874] [General] [answered]
![20201106_133134A](https://user-images.githubusercontent.com/4570236/98368663-02abc100-2038-11eb-8adb-1b51af64d3e9.gif)
![20201106_133134B](https://user-images.githubusercontent.com/4570236/98368681-08a1a200-2038-11eb-9bed-b33562927bd6.gif)

https://paste.the-compiler.org/view/db355961 as version info

Flickering severity correlates with activity on screen. No specific pages, happening on all of them.

Interesting fact: I tried recording this w/ ffmpeg first (x11grab) and the flickering was not visible on the recording, even if setting a high framerate.

I'd really appreciate any help to resolve this. I don't thing this is a bug in qutebrowser, just wanted to get some help here.

**Comment by grault** [comment]
![20201106_140356](https://user-images.githubusercontent.com/4570236/98369465-4a7f1800-2039-11eb-8795-42894a847e74.gif)

^^ this is a severe case, I just submitted this discussion and probably bc of the gifs playing, it flickers even more

**Comment by wasamasa** [comment, marked as answer]
I've added `c.qt.force_software_rendering = 'software-opengl'` to my config file for similar reasons, although it manifested itself as lack of rerendering (until forcing a repaint by a key/mouse event) when switching workspaces in i3.

**Reply by grault** [reply]
`'software-opengl'` did not work, but `'qt-quick'` did, so I marked your answer an answer. Thanks a lot for pointing to opengl and `qt.force_software_rendering`!
